examples/ml-pipelines-node-classification.ipynb

```diff
@@ -0,0 +1,1540 @@
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   nodeId  propertyValue  subject\n",
+       "0       0            100      100\n",
+       "1       1              1        1\n",
+       "2       2              2        2\n",
+       "3       3              2        2\n",
+       "4       4              3        3\n",
+       "5       5              3        3\n",
+       "6       6              4        4\n",
+       "7       7            100      100\n",
+       "8       8            100      100\n",
+       "9       9              4        4"
+      ]
+     },
+     "execution_count": 154,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Retrieve node information from Neo4j using the node IDs from the prediction result\n",
+    "nodes = gds.util.asNodes(predicted_fastrp.nodeId.to_list())\n",
+    "\n",
+    "# Create a new DataFrame containing node IDs along with node properties\n",
+    "nodes_df = pd.DataFrame([(node.id, node[\"subject\"]) for node in nodes], columns=[\"nodeId\", \"subject\"])\n",
+    "\n",
+    "# Merge with the prediction result on node IDs, to check the predicted value\n",
+    "# against the original subject\n",
+    "#\n",
+    "# NOTE: This could also be replaced by just appending `node[\"subject\"]` as a\n",
+    "# Series since the node order would not change, but a proper merge (or join)\n",
+    "# is clearer and less prone to errors.\n",
+    "predicted_fastrp.merge(nodes_df, on=\"nodeId\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Z27MnchLr9mi"
+   },
+   "source": [
+    "As we can see, the prediction for all the holdout nodes is accurate."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "UdEVypaiMc5d"
+   },
+   "source": [
+    "## Write result back to Neo4j"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "aKuUQwxbPOaV"
+   },
+   "outputs": [],
+   "source": [
+    "model_fastrp.predict_write(\n",
+    "    G,\n",
+    "    writeProperty=\"predictedSubject\",\n",
+    "    modelName=\"cora-pipeline-model-fastrp\",\n",
+    "    predictedProbabilityProperty=\"predictedProbabilities\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "plRCiikGOofd"
+   },
+   "source": [
+    "## Cleanup\n",
+    "\n",
+    "When the graph, the model and the pipeline are no longer needed, they should be dropped to free up memory:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "RFWzcrX3ORXI"
+   },
+   "outputs": [],
+   "source": [
+    "model.drop()\n",
+    "model_fastrp.drop()\n",
+    "node_pipeline.drop()\n",
+    "node_pipeline_fastrp.drop()\n",
+    "\n",
+    "G.drop()\n",
+    "gds.run_cypher(\"MATCH (n) DETACH DELETE n\")"
```

Comment:
We could consider to have 2 cells. One for the in-memory/GDS cleanup and another for the underlying db cleanup.
Then we could note, that on restarting the instance, the GDS parts would be dropped anyway